webtau-core/src/main/java/com/twosigma/webtau/data/traceable/TraceableValue.java

```diff
@@ -44,6 +45,8 @@ public void updateCheckLevel(CheckLevel newCheckLevel) {
             return;
         }
 
+        newCheckLevel = isAlwaysFuzzyPassedTracing.get() ? CheckLevel.FuzzyPassed : newCheckLevel;
```

Comment:
Not generally a fan of modifying parameters.  Can we change this to use/create a new var?

Reply:
Makes sense. Also removed unused modifier lurking there.